package _14_java;

In Java does the finally block get executed if we insert a return statement inside the try block of a try catch finally

Yes
Exceptions where the finally does not get executed
- VM exits during try catch execution
- Thread executing try catch gets killed
